adal/src/androidTest/java/com/microsoft/aad/adal/AcquireTokenSilentHandlerTest.java

```diff
@@ -46,6 +50,10 @@
 import javax.crypto.spec.PBEKeySpec;
 import javax.crypto.spec.SecretKeySpec;
 
+import static android.R.attr.minSdkVersion;
```

Comment:
@iambmelt I addressed the comment. Could you approve this PR if everything looks good to you?